src/models/user.model.test.ts

```diff
@@ -0,0 +1,276 @@
+
+      //WHEN
+      mapper.scan = jest.fn(() => scanIterator)
+
+      //THEN
+      const respObtained = await User.getAll()
+
+      expect(respObtained).not.toBe(null)
+      expect(respObtained.users.length).toBe(2)
+      expect(respObtained.users).toBe(usersExpected)
+    })
+
+    it('Get next 2 users successfully', async () => {
+
+      //GIVEN       
+      const usersExpected = [new User(
+        'name',
+        new Date(2007, 2, 3),
+        'Tv 4 # 54-56',
+        'test user',
+        'www.testuser.com'
+      ), new User(
+        'name2',
+        new Date(2007, 2, 3),
+        'Tv 42 # 54-56',
+        'test user2',
+        'www.testuser2.com'
+      )]
+
+      usersExpected[0].id = '12345'
+      usersExpected[1].id = '54321'
+
+      const responseExpected = {
+        lastEvaluatedId: usersExpected[1].id,
+        users: usersExpected,
+      }
+
+      const pagesFunction = jest.fn(() => {
+        return {
+          next: () => {
+            return { value: usersExpected }
+          },
+          lastEvaluatedKey: usersExpected[1]
+        }
+      })
+
+      const scanIterator = { pages: pagesFunction, paginator: { paginator: { limit: 2, _count: 0, _scannedCount: 0, lastResolved: {}, client: { config: { credentials: { expired: false, expireTime: null, refreshCallbacks: [], accessKeyId: "key" }, credentialProvider: { providers: [null, null, null, null, null, null, null], resolveCallbacks: [] }, region: "us-central-1", logger: null, apiVersions: {}, apiVersion: null, endpoint: "http://localhost:8000", httpOptions: { timeout: 120000 }, maxRedirects: 10, paramValidation: true, sslEnabled: true, "s3ForcePathStyle": false, "s3BucketEndpoint": false, "s3DisableBodySigning": true, "s3UsEast1RegionalEndpoint": "legacy", computeChecksums: true, convertResponseTypes: true, correctClockSkew: false, customUserAgent: " dynamodb-data-mapper-js/0.4.0", "dynamoDbCrc32": true, systemClockOffset: 0, signatureVersion: null, signatureCache: true, retryDelayOptions: {}, useAccelerateEndpoint: false, clientSideMonitoring: false, endpointCacheSize: 1000, hostPrefixEnabled: true, stsRegionalEndpoints: "legacy", accessKeyId: "key", secretAccessKey: "key" }, endpoint: { protocol: "http:", host: "localhost:8000", port: 8000, hostname: "localhost", pathname: "/", path: "/", href: "http://localhost:8000/" }, _events: { apiCallAttempt: [null], apiCall: [null] }, _clientId: 1 }, nextRequest: { TableName: "users", Limit: 2, ExclusiveStartKey: { id: { S: "af4f510f-abe3-4f0a-ae55-79f6c7dfda7a" } } } }, lastResolved: {}, itemSchema: { id: { type: "String", keyType: "HASH" }, name: { type: "String" }, dob: { type: "Date" }, address: { type: "String" }, description: { type: "String" }, createdAt: { type: "Date" }, updateAt: { type: "Date" }, imageUrl: { type: "String" } } }, _count: 0, lastResolved: {}, pending: [] } as any as ScanIterator<User>
```

Comment:
This line has more than 1500 characters, wouldn't you consider prettier an option to avoid lines like this?

Reply:
Yes completely agree. The reason I didn't use prettier was to time constraints. But I just implemented now in the solution